src/components/victory-chart/victory-chart.jsx

```diff
@@ -233,46 +211,39 @@ export default class VictoryChart extends React.Component {
 
   getNewChildren(props, childComponents, baseStyle) {
     const calculatedProps = this.getCalculatedProps(props, childComponents);
-
-    const getTransitionProps = Transitions.getTransitionPropsFactory(
-      childComponents,
-      this.state,
-      this.props.animate,
-      (newState) => this.setState(newState)
-    );
-
+    const getAnimationProps = Wrapper.getAnimationProps.bind(this);
     return childComponents.map((child, index) => {
-      const transitionProps = getTransitionProps(child.props, child.type, index);
       const style = defaults({}, child.props.style, {parent: baseStyle.parent});
       const childProps = this.getChildProps(child, props, calculatedProps);
-
-      return React.cloneElement(child, defaults({
+      const newProps = defaults({
+        animate: getAnimationProps(props, child, index),
         height: props.height,
         width: props.width,
         padding: Helpers.getPadding(props),
         ref: index,
         key: index,
         standalone: false,
         style
-      }, transitionProps, childProps));
+      }, childProps);
+      return React.cloneElement(child, newProps);
     });
   }
 
   render() {
-    const style = this.getStyles(this.props);
-    const childComponents = this.state && this.state.nodesWillExit ?
-      ChartHelpers.getChildComponents(this.state.oldProps, defaultAxes) :
-      ChartHelpers.getChildComponents(this.props, defaultAxes);
+    const props = this.state && this.state.nodesWillExit ?
+      this.state.oldProps : this.props;
```

Comment:
This change may effect how animations would change from one render to the next.  If the `animation` prop changes, this will now use `oldProps.animation` instead of `this.props.animation`.